Make it possible to dump concepts (#1196)
tenzir/vast: doc/cli/vast-count.md

> The `count` command counts a subset of data according to a given query
> expression.  This is easiest explained on an example:

Reviewer: ```suggestion
The `count` command counts the number of events that a given query
expression yields.  For example:
```

```bash
vast count ':addr in 192.168.0.0/16'
```

> The above command prints the amount of events in the database which have an
> address field in the subnet `192.168.0.0/16`.

Reviewer: ```suggestion
This prints the number of events in the database that have an
address field in the subnet `192.168.0.0/16`.
```

An optional `--estimate` flag skips the candidate checks, i.e., asks only the

> index and does not verify the hits against the database. This is a much quicker
> operation and very useful in scenarios where an upper bound is required only.

Reviewer: ```suggestion
index and does not verify the hits against the database. This is a faster
operation and useful when an upper bound suffices.
```
